docs/advocate-kit/why.md

```diff
@@ -0,0 +1,40 @@
+# Why did we create these Principles?
+
+This page describes the reasoning, intentions, and thoughts behind the Principles of Authentic Participation.
+While it does not talk about the Principles specifically, it adds background context and history for how the initial work on the Principles began.
+
+
+## History
+
+The Principles of Authentic Participation were derived at the [Sustain Summit 2020](https://blog.jwf.io/2020/02/sustain-oss-2020-quick-rewind/) event on 30 January 2020 in Brussels, Belgium.
+There, [Duane O'Brien](https://github.com/DuaneOBrien) and others facilitated discussion groups loosely focused on corporate accountability in the context of open source.
+From these discussion groups, the following four goals emerged based on discussion topics from the day:
+
+1. Set and publish a goal for open source contribution relative to value capture
+1. Adhere to the Principles of Authentic Participation
+1. Publish documentation of open source policies, processes, and project governance
+1. Well-defined reporting process that is publicly available
+
+From these four goals at Sustain Summit 2020, two went on to become focuses of an _Accountability & Transparency_ Working Group affiliated with the Sustain Summit.
+What you are reading now are the deliverables created by the group focused on the Principles of Authentic Participation.
+
+
+## Common struggles
+
+How did the discussion groups in Brussels agree that these Principles of Authentic Participation were important?
+First, consider that the original Sustain Summit 2020 discussion groups aimed to identify positive _and_ negative characteristics of corporate participation in open source.
+
+However, the Sustain Summit revealed a common pressure point.
+Folks with backgrounds in humanitarian, civic, and non-profit sectors also shared similar problems with their organizations exploring open source.
```

Comment:
Reading this, not sure if I'm just missing something obvious but I'm not really sure what these similar problems are.

Reply:
Probably could use more concrete examples here. I can solicit feedback from the Working Group on this. Probably leaning a bit on @DuaneOBrien.